src/components/Input/QtySelector/QtySelector.js

```diff
@@ -161,15 +161,27 @@ class QtySelector extends Component {
 
     return (
       <Container>
-        <Button onClick={this.decrement} disabled={disabled || min === value}>
+        <Button
+          onClick={this.decrement}
+          disabled={disabled || min === value}
+          aria-label="Decrease Quantity"
```

Comment:
Should probably take it as a prop, given we need to translate the label for other locales (en-US, fr-CA etc)

Reply:
Thank you. I agree. I took them as props now with the "Decrease Quantity" and "Increase Quantity" as the default texts. I also removed the `aria-disabled` attr as the existing `disabled` attr has already had the same purpose.